pkg/remediator/crashloopbackoffrescheduler_test.go

```diff
@@ -64,125 +68,84 @@ func (suite *TestCrashLoopBackOffReschedulerSuite) SetupTest() {
 		},
 		Status: corev1.PodStatus{
 			InitContainerStatuses: []corev1.ContainerStatus{
-				corev1.ContainerStatus{
-					RestartCount: 0,
-					State: corev1.ContainerState{
-						Running: &corev1.ContainerStateRunning{},
-					},
-				},
-			},
```

Comment:
I would prefer to keep 2 pods in the list so as to check it doesn't touch healthy pod.

Reply:
there are separate tests for healthy pods

Reply:
... that's the beauty of it, 1 test for each behavior ... much simpler to reason about/debug :)